fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/query/expression/StringExpNodeVisitor.java

```diff
@@ -0,0 +1,406 @@
+    }
+
+    @Override
+    public String or(String left, String right) {
+        StringBuilder result = new StringBuilder();
+        result.append(left);
+        if (pretty) {
+            result.append(NEWLINE).append("         "); // 10 spaces
+        }
+        result.append(" OR ");
+        result.append(right);
+        return result.toString();
+    }
+
+    @Override
+    public String not(String exp) {
+        StringBuilder result = new StringBuilder();
+        if (pretty) {
+            result.append(NEWLINE).append("        "); // 9 spaces
+        }
+        result.append("NOT ");
+        result.append(exp);
+        return result.toString();
+    }
+
+    @Override
+    public String exists(String selectString) {
+        StringBuilder result = new StringBuilder();
+        result.append("EXISTS (");
+        result.append(selectString);
+        result.append(")");
+        return result.toString();
+    }
+
+    @Override
+    public String notExists(String selectString) {
+        StringBuilder result = new StringBuilder();
+        result.append("NOT EXISTS (");
+        result.append(selectString);
+        result.append(")");
+        return result.toString();
+    }
+
+    @Override
+    public String eq(String left, String right) {
+        StringBuilder result = new StringBuilder();
+        result.append(left);
+        result.append(" = ");
+        result.append(right);
+        return result.toString();
+    }
+
+    @Override
+    public String neq(String left, String right) {
+        StringBuilder result = new StringBuilder();
+        result.append(left);
+        result.append(" != ");
+        result.append(right);
+        return result.toString();
+    }
+
+    @Override
+    public String gt(String left, String right) {
+        StringBuilder result = new StringBuilder();
+        result.append(left);
+        result.append(" > ");
+        result.append(right);
+        return result.toString();
+    }
+
+    @Override
+    public String gte(String left, String right) {
+        StringBuilder result = new StringBuilder();
+        result.append(left);
+        result.append(" >= ");
+        result.append(right);
+        return result.toString();
+    }
+
+    @Override
+    public String lt(String left, String right) {
+        StringBuilder result = new StringBuilder();
+        result.append(left);
+        result.append(" < ");
+        result.append(right);
+        return result.toString();
+    }
+
+    @Override
+    public String lte(String left, String right) {
+        StringBuilder result = new StringBuilder();
+        result.append(left);
+        result.append(" <= ");
+        result.append(right);
+        return result.toString();
+    }
+
+    @Override
+    public String literal(String value) {
+        StringBuilder result = new StringBuilder();
+        result.append("'");
+        result.append(value);
+        result.append("'");
+        return result.toString();
+    }
```

Comment:
What happens when the value contains a `'`.... that's probably when we should encode it.

Reply:
Literals are already encoded at this point.